classes/02class/exercises/c02-network06/frdvo/README.md

```diff
@@ -0,0 +1,72 @@
+# c02-network06
+
+## Commands Execution Output
+
+- Commands for creating the NAT gateway:
+```
+#Allocate an Elastic IP
+aws ec2 allocate-address
+
+#Create the NAT Gateway, allocate to public-b
+aws ec2 create-nat-gateway --allocation-id eipalloc-0438e982d1be851e3 --subnet-id subnet-0dbb2205880974071
+
+#Tag Nat Gateway
+aws ec2 create-tags --resources nat-02330e26283489e20 --tags Key=Name,Value=devopsacademy-ngw
+
+#Delete NAT Gateway
+aws ec2 delete-nat-gateway --nat-gateway-id nat-02330e26283489e20
+
+#Delete IP Address
+aws ec2  release-address --allocation-id eipalloc-0438e982d1be851e3
+
+
+Add your commands and their outputs here
+```bash
+ 🐳 :: 📦 2  📑 :: 1   👉 /home/fer/repos/academy/classes/02class/exercises/c02-network06  
```

Comment:
Same comment as the other PR about formatting.
Can you add the outputs just below the commands above and remove the emojis and terminal decoration?

Otherwise it looks good 👍